backend/src/common/utils/cryptrHelper.ts

```diff
@@ -0,0 +1,8 @@
+import Cryptr from 'cryptr';
+import { env } from '../../env';
+
+const cryptr = new Cryptr(env.app.secret);
+
+export const encryptString = (data: string) => cryptr.encrypt(data);
+
+export const decryptString = (data: string) => cryptr.decrypt(data);
```

Comment:
```suggestion
export const decrypt = (data: string) => cryptr.decrypt(data);
```

it's clear that you encrypt string by type of the passed parameter